lib/init/index.js

```diff
@@ -12,42 +13,45 @@ var languages = {
 var exportables = {};
 
 // Initialize the directory given the various options
-exportables.initProject = (opts) => {
-
+exportables.createNewProject = (options) => {
   // Set a default directory if one was not provided
-  opts.directory = opts.directory || path.resolve('.');
+  options.directory = options.directory || path.resolve('.');
 
   // Detect the requested language
-  var lang = exportables.resolveLanguage(opts);
+  var lang = exportables.resolveLanguage(options.lang);
 
   // If a language could not be detected
   if (lang === null) {
-    // Return an error
     return Promise.reject(new Error('Unrecognized language selection.'));
   } else {
     // Otherwise generate a project in that language
-    return lang.generateProject(opts);
+    return lang.generateProject(options);
   }
 };
 
 // Determine the langauge to initialize the project with
-exportables.resolveLanguage = (opts) => {
+exportables.resolveLanguage = (input) => {
 
   // If somehow a language option wasn't provided
-  if (!opts.lang) {
-    // Return JS as default
-    return languages['js'];
+  if (!input) {
+    return languages.js;
+  }
+
+  input = String(input).toLowerCase();
```

Comment:
Is there a case where `input` wouldn't be a `String`? I'm guessing it's coerced to one just in case. 

If it's not a `String`, could we `return null` earlier?